src/components/app/header.tsx

```diff
@@ -11,10 +11,7 @@ import {useRouter} from 'next/router'
 import {useTheme} from 'next-themes'
 import useCio from '../../hooks/use-cio'
 import {Form, Formik} from 'formik'
-import LevelUpCTA from '../survey/level-up-cta'
-
-const ACCOUNT_LINK_ENABLED =
-  process.env.NEXT_PUBLIC_FEATURE_ACCOUNT_LINK_IN_HEADER === 'true'
+import OnlinePresenceCTA from '../survey/online-presence-cta'
```

Comment:
I just cut and pasted this into a new file, it will probably make a nice common component someday